fpna-template/Files/AmountType.ipynb

```diff
@@ -0,0 +1,100 @@
+{
+    "metadata": {
+        "authors": [],
+        "id": "mGne7GUOsEmlryy75P7L6Q",
+        "kernelspec": {
+            "display_name": "Formula Framework",
+            "language": "C#",
+            "name": "C#"
+        },
+        "language_info": {
+            "file_extension": ".cs",
+            "mimetype": "text/plain",
+            "name": "C#"
+        }
+    },
+    "nbformat": 4,
+    "nbformat_minor": 5,
+    "cells": [
+        {
+            "cell_type": "markdown",
+            "source": [
+                "# Amount Type"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "var amountTypes = @\"@@AmountType",
```

Comment:
We might need to be careful with this type of inheritance, as SystemNames of derived types could clash, as they are saved under the same base type.

We can discuss this in a modelling meeting, because we could consider having the discriminator as a property.

Reply:
for now I have renamed this to EarningsAmountType, which inherits from AmountType. Still the SystemNames may clash, so perhaps we need to think of a better concept. adding a TODO to the DataModel